ccmlib/scylla_repository.py

```diff
@@ -316,7 +316,10 @@ def setup_scylla_manager(scylla_manager_package=None):
                     mkdir -p dist/etc
                     mkdir schema
                     cp -r ./etc/scylla-manager/* ./dist/etc/
-                    cp -r ./etc/scylla-manager/cql ./schema/
+                    if [ -d "./etc/scylla-manager/cql" ]
```

Comment:
What may happen if schema won't be copied? Does it has affect?

Reply:
Not at all, according to michal they moved it into the binary. That's also why we don't config the migrate_dir parameter anymore